pkgs/applications/window-managers/dwm/git.nix

```diff
@@ -1,16 +1,18 @@
-{ stdenv, fetchgit, libX11, libXinerama, libXft, patches ? [], conf ? null }:
+{ stdenv, fetchgit, libX11, libXinerama, libXft, writeText, patches ? [ ]
+, conf ? null }:
 
-let
-  name = "dwm-git-20180602";
-in
+with stdenv.lib;
```

Comment:
Pelase add the with where you need it and don't do it over the entire file.

Reply:
Sandro, got it, updated.